mucommander-core/src/main/java/com/mucommander/ui/action/impl/ShowFilePopupMenuAction.java

```diff
@@ -23,46 +23,55 @@
 
 import javax.swing.KeyStroke;
 
-import com.mucommander.commons.file.util.FileSet;
+import com.mucommander.commons.file.AbstractFile;
+import com.mucommander.core.desktop.DesktopManager;
+import com.mucommander.text.Translator;
 import com.mucommander.ui.action.AbstractActionDescriptor;
 import com.mucommander.ui.action.ActionCategory;
 import com.mucommander.ui.action.ActionDescriptor;
+import com.mucommander.ui.action.ActionProperties;
+import com.mucommander.ui.dialog.InformationDialog;
 import com.mucommander.ui.main.MainFrame;
 import com.mucommander.ui.main.menu.TablePopupMenu;
 import com.mucommander.ui.main.table.FileTable;
+import com.mucommander.ui.main.table.FileTableModel;
 
 
 /**
- * This action shows the popup menu for currently selected file
- *
- * @author Miroslav Oujesky
+ * This action shows the popup menu for currently selected file.
+ * Exactly the same popup menu appears also when right-clicking on file/folder.
  */
-public class ShowFilePopupMenuAction extends SelectedFilesAction {
+public class ShowFilePopupMenuAction extends SelectedFileAction {
 
     public ShowFilePopupMenuAction(MainFrame mainFrame, Map<String, Object> properties) {
         super(mainFrame, properties);
+        setEnabled(DesktopManager.canOpenInFileManager());
```

Comment:
why do we need this part? it's ok to present the popup menu without the RevealInDesktopAction operation in this case, no?